consensus/types/src/payload_attestation_data.rs

```diff
@@ -0,0 +1,27 @@
+use crate::test_utils::TestRandom;
+use crate::*;
+use serde::{Deserialize, Serialize};
+use ssz_derive::{Decode, Encode};
+use test_random_derive::TestRandom;
+use tree_hash_derive::TreeHash;
+
+#[derive(
+    TestRandom, TreeHash, Debug, Clone, PartialEq, Eq, Encode, Decode, Serialize, Deserialize, Hash,
+)]
+#[cfg_attr(feature = "arbitrary", derive(arbitrary::Arbitrary))]
+#[context_deserialize(ForkName)]
+pub struct PayloadAttestationData {
+    pub beacon_block_root: Hash256,
+    pub slot: Slot,
+    pub payload_present: bool,
+}
+// todo(eip-7732): Mark's implementation  has PayloadStatus as an enum, but spec calls for a bool. Need to clarify this.
```

Comment:
@ethDreamer, just flagging this spec change from your initial impl, [spec](https://ethereum.github.io/consensus-specs/specs/_features/eip7732/beacon-chain/#payloadattestationdata)